src/library/Book.java

```diff
@@ -0,0 +1,68 @@
+package library;
+
+
+import java.util.Objects;
+
+public class Book implements Comparable<Book> {
+
+    private int id;
+    private String title;
+    private String genre;
+
+    public Book(int id, String title, String genre) {
+        this.id = id;
```

Comment:
let's generate ID by ourself (every new book should have ID greater than previous book on 1)

Reply:
Done